FreydCategoriesForCAP/gap/CategoryOfGradedRowsAndColumns/CategoryOfGradedColumns.gi

```diff
@@ -30,8 +30,10 @@ InstallMethod( CAPCategoryOfGradedColumns,
 
       # set its properties
       SetIsAdditiveCategory( category, true );
-      SetIsStrictMonoidalCategory( category, true );
-      SetIsRigidSymmetricClosedMonoidalCategory( category, true );
+      if IsCommutative( homalg_graded_ring ) then
```

Comment:
Here I would draw attention to this issue
https://github.com/homalg-project/homalg_project/issues/191

Reply:
Thanks for this pointer. I have changed the code such that it test if the underlying non-graded ring is commutative.